generalized_linear_regression/generalized_linear_regression.md

```diff
@@ -0,0 +1,528 @@
+
+Something like medication dose could be continuous or categorical depending on how it's used. 
+If patients are restricted to categorical levels of dose (e.g. high, med, or low dose groups) then it would be categorical. 
+But if it's something where their dose might theoretically be anywhere on the scale, then it's continuous.
+
+<div class = "learn-more">
+<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>
+
+For an excellent review of continuous vs. categorical variables, see the [Khan Academy video on discrete and continuous random variables](https://www.khanacademy.org/math/statistics-probability/random-variables-stats-library/random-variables-discrete/v/discrete-and-continuous-random-variables).
+
+</div>
+
+### Boundedness
+
+A variable is **unbounded** if it can theoretically extend from negative infinity at the low end to positive infinity at the high end. 
+We often have variables that aren't truly unbounded, but they're close enough for all practical purposes -- for example, IQ can't technically be negative, but in practice no measured IQs are close to 0 so the data stop naturally before they hit the scale boundary at 0. 
+IQ is generally treated as unbounded statistically.
+
+Something like a probability or proportion is bounded at 0 and 1; it's not possible to have a negative probability, or one over 1. 
+
+A variable like age might be unbounded or bounded depending on what ages you're studying. 
+In a young pediatric population, age is bounded at 0 because you can't have any negative ages and a distribution of infant ages will get cutoff at that lower bound. 
+But if you're studying adults, then age might be functionally unbounded in your dataset.
+
+### Quiz: Types of Outcome Variables
+
+Dr. Jemison wants to run a model predicting length of stay (calculated from timestamps in the electronic health record) for patients at her hospital. How would you describe her outcome variable?
+
+[( )] continuous and unbounded
+[(X)] continuous, but not unbounded
+[( )] unbounded, but not continuous
+[( )] neither continuous nor unbounded
+***
+<div class = "answer">
+
+Length of stay is continuous because it can be any value in its range -- a patient might have a measured length of stay of 2 days, or 2.5 days, or 2 days 3 hours and 12 minutes, etc. 
+It's not discrete categories of time, it's a continuous scale.
+
+But it's not unbounded! 
+Although there's no theoretical upper limit on length of stay, it would be impossible to have a length of stay less than 0 days, so the variable is bound at zero.
+And unless Dr. Jemison is studying a particular population that all have unusually long hospital stays, it's likely that the data will be effectively cutoff at 0 -- there will plenty of patients with a length of stay of 3 days, and 2 days, and 1 day, but then of course nothing below zero.
```

Comment:
I've tried to include a few examples in the module to emphasize that boundedness is sometimes a subjective judgment -- e.g. I talk about IQ as well, which is also technically bound at 0, but in reality never gets close to that cutoff so it behaves as though it were unbounded, and we generally treat it so statistically. I also talk about being potentially bounded, if you're looking at a pediatric population where the 0 cutoff will affect the data, but unbounded if you're looking at an older population where the youngest participants are well above 0. 
It's a slippery issue, so I do want to make sure to treat it thoroughly. This mention is the third example, though, so maybe that's enough?

Reply:
What you've said here makes sense to me, and I agree with the importance of giving this a thorough treatment but I'm still confused by the sentence "unless Dr. Jemison is studying a particular population that all have unusually long hospital stays, it's likely that the data will be effectively cutoff at 0" -- why does the fact that the stays will be short in length indicate that the data will be effectively cutoff at zero? 

Maybe I just got stuck in a weird thought loop here that explains my confusion, but it seems to me that the fact that its cutoff at 0 (as a lower bound) is already explained by the fact that you can't have a negative number of days spent in the hospital. This is true regardless of if she's studying a population with relatively short or long duration of stay. 

If she's studying mostly patients with a short duration of stay, then maybe there's some effective upper bound -- which is the conclusion the first half of that sentence + the portion after the `--` brings me to. Am I on track to say that's what you're intending to get at? 
Where I get caught up is in the middle of the sentence where it says that the data is bound at zero effectively, because she's not looking at pts with abnormally long lengths of stay. 

Another way to put this is that even if she was studying patients with very long duration of stay (1 month or more, say), the data would still be cut off at zero at the lower bound, right?

Reply:
Ah, oh, I see! I think maybe the culprit is my choice of the word "effectively" . Length of stay data are *technically* always cutoff at zero, as you note, because negative stays make no sense. But in a population that all have long stays, the data aren't cutoff anywhere in a way that means anything practically, they just peter out at the high and low ends. So maybe data from a long term care facility, where all the patients are there at least a month or two, some maybe for years, the data are effectively unbounded. Even though technically there is a bound at 0, it's just that we'd never hit it so the fact that it's technically there doesn't matter statistically. 
But most length of stay data are bounded at zero in a way that does matter, because there tend to be plenty of observations right up until that bound at 0. 
Does this word change help?

```suggestion
And unless Dr. Jemison is studying a particular population that all have unusually long hospital stays, it's likely that the data will be noticeably cutoff at 0 -- there will plenty of patients with a length of stay of 3 days, and 2 days, and 1 day, but then of course nothing below zero.
```

Reply:
AH okay, I see where you were going with this now!!!! 

Maybe situating it in relation to some of those examples where we can treat it as unbounded would help? 

Here's my iteration! Does this still emphasize the correct point? 

```suggestion
And unless Dr. Jemison is studying a particular population that all have unusually long hospital stays, it's likely that the data will actually run right up to the boundary at 0 -- there will plenty of patients with a length of stay of 3 days, and 2 days, and 1 day, and then of course nothing below zero. This is why we would want to treat it as bounded statistically, unlike IQ, which you may recall we can treat as unbounded despite also not allowing for negative values, because in practice no IQ measurements are ever close to the scale boundary of 0. 
```

Reply:
I like this approach! What do you think about keeping the explanation closer to the current example, though, as below?

```suggestion
And unless Dr. Jemison is studying a particular population that all have unusually long hospital stays, it's likely that the data will actually run right up to the boundary at 0 -- there will plenty of patients with a length of stay of 3 days, and 2 days, and 1 day, and then of course nothing below zero.
That's why length of stay would have to be treated as bounded in this example.

If Dr. Jemison was in fact studying a population with very long hospital stays (perhaps patients with a condition that nearly always requires hospital stays of weeks or months) and none of the lengths of stay in the data were close to 0, then it could be treated as unbounded -- even though there would still technically be a lower bound at 0, that doesn't matter statistically if the data don't approach it (just like in the IQ example earlier). 
```

Reply:
Love it!!!!